othertasks/concurrencyExample.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+)
+
+var filter = ".com"
+
+var wg sync.WaitGroup
+
+// FindCorrectString filters inputed url.
+func FindCorrectString(input string, output chan string) {
+	if strings.Contains(input, filter) {
+		output <- input
+	}
+	defer wg.Done()
+}
+
+// Finder returns urls that contain ".com".
```

Comment:
Realy, looks lite it print them, `-er` - is a sign of interface, try to avoid it in the name of function
Filtering - is more suitable for that case